plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -664,16 +664,17 @@ public void testVarbinary()
     @Test
     public void testDateMapping()
     {
-        DateTests(zone -> trinoCreateAsSelect("nl_date_" + zone));
+        dateTests(zone -> trinoCreateAsSelect("date_" + zone));
+        dateTests(zone -> trinoCreateAndInsert("date_" + zone));
     }
 
     @Test
     public void testDateReadMapping()
     {
-        DateTests(zone -> oracleCreateAndInsert("nl_read_date_" + zone));
+        dateTests(zone -> oracleCreateAndInsert("read_date_" + zone));
     }
 
-    private void DateTests(Function<String, DataSetup> dataSetup)
```

Comment:
the uppercased method name should have been flagged by checkstyle. Any clue why not? do we miss any rules?

Reply:
Add `<module name="MethodName"/>` to [airbase-checks.xml](https://github.com/airlift/airbase/blob/master/airbase-policy/src/main/resources/checkstyle/airbase-checks.xml) might help, 
it will check method name match to the default pattern '^[a-z][a-zA-Z0-9]*$'.

FYI: https://checkstyle.org/config_naming.html#MethodName

Reply:
Good idea. Are there other violations of this kind in the codebase? do you want to update airbase?

Reply:
> Are there other violations of this kind in the codebase

I do things as follows:

1. Download checkstyle jar [8.41.1](https://github.com/checkstyle/checkstyle/releases/tag/checkstyle-8.41.1) 
2. Create checkstyle xml with content as belows
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE module PUBLIC
        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
        "http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd">

<module name = "Checker">
  <property name="charset" value="UTF-8"/>
  <module name="TreeWalker">
	<module name="MethodName" />
  </module>
</module>
```
3. Run cmd 
```sh
java -jar ${checkstyle-jar-location} \
  -c ${checkstyle-xml-location} \
  -x ".*/target/" \
  ${trino-root-location}
```

And I got errors as belows:
```txt
Starting audit...
[ERROR] /Users/brandboat/Code/trino/core/trino-main/src/test/java/io/trino/metadata/TestInternalBlockEncodingSerde.java:41:17: Name 'BlockRoundTrip' must match pattern '^[a-z][a-zA-Z0-9]*$'. [MethodName]
[ERROR] /Users/brandboat/Code/trino/core/trino-main/src/main/java/io/trino/operator/scalar/VarbinaryFunctions.java:318:25: Name 'spookyHashV2_32' must match pattern '^[a-z][a-zA-Z0-9]*$'. [MethodName]
[ERROR] /Users/brandboat/Code/trino/core/trino-main/src/main/java/io/trino/operator/scalar/VarbinaryFunctions.java:328:25: Name 'spookyHashV2_64' must match pattern '^[a-z][a-zA-Z0-9]*$'. [MethodName]
[ERROR] /Users/brandboat/Code/trino/core/trino-main/src/main/java/io/trino/operator/scalar/LuhnCheckFunction.java:32:27: Name 'LuhnCheck' must match pattern '^[a-z][a-zA-Z0-9]*$'. [MethodName]
[ERROR] /Users/brandboat/Code/trino/core/trino-main/src/main/java/io/trino/operator/scalar/MathFunctions.java:1189:26: Name 'NaN' must match pattern '^[a-z][a-zA-Z0-9]*$'. [MethodName]
[ERROR] /Users/brandboat/Code/trino/plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreClient.java:186:12: Name 'get_config_value' must match pattern '^[a-z][a-zA-Z0-9]*$'. [MethodName]
[ERROR] /Users/brandboat/Code/trino/client/trino-client/src/test/java/io/trino/client/auth/external/TestHttpTokenPoller.java:179:17: Name 'testTokenReceivedRetriesUntilHTTP_OK' must match pattern '^[a-z][a-zA-Z0-9]*$'. [MethodName]
Audit done.
Checkstyle ends with 7 errors.
```

> do you want to update airbase?

Sure, I'm glad to do it. Since module `MethodaName` default not allow `_` show up in method name, should we add `_` char to white list ?

Reply:
we can remove _ from testTokenReceivedRetriesUntilHTTP_OK ( -> testTokenReceivedRetriesUntilHttpOk), 
but i don't have a good naming idea for spookyHashV2_32, spookyHashV2_64, so yes, let's allow the underscore

Reply:
Filed https://github.com/airlift/airbase/pull/264